models/profiles.yaml

```diff
@@ -154,7 +154,7 @@ var_groups:
           description: It shows the average units purchased in each transaction. (Total units in each transaction/Total transactions). Includes only those transactions where the total price (from column current_total_price) is greater than zero. So, the feature exclude transactions with 100% off, replacement products etc that may result in the total_price being equal to zero.
       - entity_var:
           name: avg_transaction_value
-          select: avg( total_price_usd )
+          select: avg( CAST(total_price_usd AS {{warehouse.DataType("float")}}) )
```

Comment:
Why cast it to float ? Is some warehouse automatically converting it to int ?

Reply:
it's initially stored as a string if we are taking input from csv. BigQuery is very particular about data types, and it won't automatically convert a string to a numeric type for aggregation functions like other warehouses.